modules/oracle-xe/src/main/java/org/testcontainers/containers/OracleContainer.java

```diff
@@ -22,8 +22,7 @@
     private String password = "oracle";
 
     private static String resolveImageName() {
-        String image = TestcontainersConfiguration.getInstance()
-            .getProperties().getProperty("oracle.container.image");
+        String image = TestcontainersConfiguration.getInstance().getOracleImage();
```

Comment:
shouldn't it be `DEFAULT_IMAGE`, similar to `KafkaContainer` and others?

Reply:
Oracle is weird, because we (currently) have no default image to point to. I think perhaps we should resolve _that_, and do so in a separate change.

Reply:
oh, right! I completely forgot about it 😅